### ISSUE TITLE
log_proxy_wrapper fails with -O, -E, and -F, missing new options.

### ISSUE THREAD

**Issue by tomalok** [metwork-framework/log_proxy#25]
What happened to `--stdout`, `--stderr`, and `--fifo-tmp-dir`?  I'm not seeing it in v0.4.0's `log_proxy_wrapper --help`...
```
$ /usr/bin/log_proxy_wrapper -F /var/log/log_proxy -O /var/log/docker.log -s 10485760 -t 0 -- /usr/bin/dockerd
Usage:
  log_proxy_wrapper [OPTION…] -- COMMAND [COMMAND_ARG1] [COMMAND_ARG2] [...] - log proxy

Help Options:
  -h, --help                Show help options

Application Options:
  -s, --rotation-size       maximum size (in bytes) for a log file before rotation (0 => no maximum, default: content of environment variable LOGPROXY_ROTATION_SIZE or 104857600 (100MB))
  -t, --rotation-time       maximum lifetime (in seconds) for a log file before rotation (0 => no maximum, default: content of environment variable LOGPROXY_ROTATION_TIME or 86400 (24H))
  -S, --rotation-suffix     strftime based suffix to append to rotated log files (default: content of environment variable LOGPROXY_ROTATION_SUFFIX or .%%Y%%m%%d%%H%%M%%S)
  -d, --log-directory       directory to store log files (default: content of environment variable LOGPROXY_LOG_DIRECTORY or current directory), directory is created if missing
  -n, --rotated-files       maximum number of rotated files to keep including main one (0 => no cleaning, default: content of environment variable LOGPROXY_ROTATED_FILES or 5)
  -m, --use-locks           use locks to append to main log file (useful if several process writes to the same file)
```

Specifically, i'm trying to use `--fifo-tmp-dir` (via the `-F` shortcut) and it's not there.

I'd also expect to be seeing the recent `--chmod`, `--chown`, and `--chgrp`, too.  Those are showing up in the help for `log_proxy` itself -- just not the the proxy...

**Comment by tomalok** [comment]
```
77be7b7da998:/# strings /usr/bin/log_proxy | grep stderr
77be7b7da998:/# strings /usr/bin/log_proxy | grep stdout
77be7b7da998:/# strings /usr/bin/log_proxy | grep fifo
fifo
if set, read lines on this fifo instead of stdin
rm-fifo-at-exit
if set, drop fifo at then end of the program (you have to use --fifo option of course)
```
I suspect [`change_options()`](https://github.com/metwork-framework/log_proxy/blob/master/src/log_proxy_wrapper.c#L22-L40) is not doing what you expect it to do.

**Comment by thefab** [comment]
thanks for this issue @tomalok

**Comment by thefab** [comment]
fixed in 0.4.1
